src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const styles = sourceString
+    .split(';')
+    .filter((style) => style.trim().length > 0);
+  const styleOBJ = {};
+
+  styles.forEach((style) => {
```

Comment:
The forEach method is appropriately used here to iterate over the styles array and populate the object.